tests/test_midpointsedges.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+import meshzoo
+
+
+def test_midpoints_edges_tri():
+    points, cells = meshzoo.rectangle_tri(a0=(0, 0), a1=(1, 1), n=2)
+    points_new, cells_new = meshzoo.insert_midpoints_edges(
+        points, cells, cell_type="triangle"
+    )
+    assert len(points_new) == 9
+    assert cells_new.shape[1] == 6
```

Comment:
Let's test the entire shape. I'd expect `cells_new.shape == (24, 6)`, is this correct?